packages/shared/styles/components/SfScrollable.scss

```diff
@@ -0,0 +1,25 @@
+@import "~simplebar/dist/simplebar.min.css";
+@import "../variables";
+
+$scrollable__max-width: 13.5rem !default;
```

Comment:
```suggestion
```
@aniamusial, why we limit width inside component, probably parent, should have responsibility for that